ckan/controllers/user.py

```diff
@@ -673,17 +667,20 @@ def dashboard(self, id=None, offset=0):
 
         c.followee_list = get_action('followee_list')(
             context, {'id': c.userobj.id, 'q': q})
+
         c.dashboard_activity_stream_context = self._get_dashboard_context(
             filter_type, filter_id, q)
-        c.dashboard_activity_stream = h.dashboard_activity_stream(
+        dashboard_activity_stream = h.dashboard_activity_stream(
```

Comment:
Normally we don't fiddle with context variables and globals because even this style is deprecated, we don't want it to break someone's customized template. However the `user/dashboard.html` template is fundamentally changed in this PR (`dashboard_activity_stream` changes type from pre-rendered HTML to a list), necessitating changes to a customized template anyway.